cmd/portal/commands/receive.go

```diff
@@ -150,7 +150,7 @@ func handleReceiveCommandRaw(version string, password string) error {
 		case errors.Is(err, io.EOF):
 			return nil
 		case errors.Is(err, file.ErrUnpackFileExists):
-			fmt.Printf("overwrite %s? [y/n] ", committer.FileName())
+			fmt.Printf("overwrite %s? [Y/n] ", committer.FileName())
```

Comment:
why not capitalize _n_ if you capitalize _y_?

Reply:
This is the de-facto standard of showing the default selection in cli programs, as I have seen. It's the same in the TUI, although there, you can also use the left/right arrow keys to change the selection, which is shown by which letter is capitalized.